crates/chain/tests/test_keychain_txout_index.rs

```diff
@@ -386,3 +386,98 @@ fn test_non_wildcard_derivations() {
         1,
     );
 }
+
+/// Check that calling `lookahead_to_target` stores the expected spks.
+#[test]
+fn lookahead_to_target() {
+    #[derive(Default)]
+    struct TestCase {
+        lookahead: u32,                      // Global lookahead value
+        external_last_revealed: Option<u32>, // Last revealed index for external keychain
+        internal_last_revealed: Option<u32>, // Last revealed index for internal keychain
+        external_target: Option<u32>,        // Call `lookahead_to_target(External, u32)`
+        internal_target: Option<u32>,        // Call `lookahead_to_target(Internal, u32)`
```

Comment:
Why not move the comments to the field's docstrings?

```suggestion
fn lookahead_to_target() {
    #[derive(Default)]
    struct TestCase {
        /// Global lookahead value
        lookahead: u32,
        /// Last revealed index for external keychain
        external_last_revealed: Option<u32>,
        /// Last revealed index for internal keychain
        internal_last_revealed: Option<u32>,
        /// Call `lookahead_to_target(External, u32)`
        external_target: Option<u32>,
        /// Call `lookahead_to_target(Internal, u32)`
        internal_target: Option<u32>,
```

Reply:
It's a styling preference. Is there a clear advantage of having doc string here?

Reply:
No :)

Reply:
> It's a styling preference. Is there a clear advantage of having doc string here?

Yes it allows your editor to tell you the purpose of the field without having to jump to the definition.

+1 for using docstrings for this.